content/en-US/api/roundRect.md

```diff
@@ -0,0 +1,122 @@
+---
+recipe: roundRect
+title: 'CanvasRenderingContext2D.roundRect()'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/roundRect
+specifications: https://html.spec.whatwg.org/#dom-context-2d-roundrect
+browser_compatibility: api.CanvasRenderingContext2D.roundRect
+---
+
+
+## Description
+
+The `roundRect` method of the `CanvasRenderingContext2D` interface adds a rounded rectangle to the 
+current sub-path.
+
+## Syntax
+
+`CanvasRenderingContext2D.roundRect(x, y, width, height, radii)`
+
+### Parameters
+<dl>
+  <dt><em>x</em></dt>
+  <dd>A double that represents the x-axis (horizontal) coordinate of the rectangle's starting point.</dd>
+
+  <dt><em>y</em></dt>
+  <dd>A double that represents the y-axis (vertical) coordinate of the rectangle's starting point.</dd>
+
+  <dt><em>width</em></dt>
+  <dd>A double that represents the rectangle's width. Positive value means the path is drawn
+  clockwise; and negative value meeans it's drawn counterclockwise and flips the rectangle
+  horizontally along `y`, that means the radius values that applied to the left corners are now
+  applied to the right.</dd>
+
+  <dt><em>height</em></dt>
+  <dd>A double that represents the rectangle's height. Positive value means the path is drawn
+  clockwise; and negative value meeans it's drawn counterclockwise and flips the rectangle
+  vertically along `x`, that means the radius values that applied to the top corners are now applied
+  to the bottom.</dd>
+
+  <dt><em>radii</em></dt>
```

Comment:
What type of structure is this? An array? A sequence? A list?

We need to rewrite the description to match the grammatical structure of the other arguments. I want to know what this is specifically before I do that.